lib/scaffold.dart

```diff
@@ -669,16 +562,7 @@ class BackdropScaffoldState extends State<BackdropScaffold>
       child: Scaffold(
         key: scaffoldKey,
         appBar: widget.appBar ??
-            AppBar(
-              title: widget.title,
-              actions: widget.iconPosition == BackdropIconPosition.action
-                  ? <Widget>[const BackdropToggleButton()] + widget.actions
-                  : widget.actions,
-              elevation: 0,
-              leading: widget.iconPosition == BackdropIconPosition.leading
-                  ? const BackdropToggleButton()
-                  : null,
-            ),
+            AppBar(elevation: 0, leading: const BackdropToggleButton()),
```

Comment:
@WieFel This will enforce having an `appBar` - should we remove the default one?

Reply:
good point actually. I would say yes, because backdrop should also be usable without an app bar

Reply:
Ok.